src/main/java/org/jboss/xavier/analytics/pojo/AdministrationMetricsModel.java

```diff
@@ -0,0 +1,77 @@
+package org.jboss.xavier.analytics.pojo;
+
+import org.apache.camel.dataformat.bindy.annotation.CsvRecord;
+import org.apache.camel.dataformat.bindy.annotation.DataField;
+
+import java.util.Date;
+
+@CsvRecord(separator = ",", crlf = "UNIX", generateHeaderColumns = true)
+public class AdministrationMetricsModel {
+
+    @DataField(pos = 1, columnName = "Analysis' owner")
+    private String owner;
+
+    @DataField(pos = 2, columnName = "Domain")
+    private String ownerDomain;
+
+    @DataField(pos = 3, columnName = "Payload name")
+    private String payloadName;
+
+    @DataField(pos = 4, columnName = "Analysis status")
+    private String analysisStatus;
+
+    @DataField(pos = 5, columnName = "Creation date (yyyy-MM-dd)", pattern = "yyyy-MM-dd")
```

Comment:
could you add the time as well please beside the date?

Reply:
I've added the time and timeZone using the format `yyyy-MM-dd HH:mm:ss z`. E.g. `2019-09-25 11:51:42 UTC`